cmd/terraform-registry/main.go

```diff
@@ -58,13 +64,15 @@ func init() {
 	flag.BoolVar(&tlsEnabled, "tls-enabled", false, "")
 	flag.StringVar(&tlsCertFile, "tls-cert-file", "", "")
 	flag.StringVar(&tlsKeyFile, "tls-key-file", "", "")
-	flag.StringVar(&storeType, "store", "", "Store backend to use (choices: github)")
+	flag.StringVar(&storeType, "store", "", "Store backend to use (choices: github, s3)")
 	flag.StringVar(&logLevelStr, "log-level", "info", "Levels: debug, info, warn, error")
 	flag.StringVar(&logFormatStr, "log-format", "console", "Formats: json, console")
 
 	flag.StringVar(&gitHubOwnerFilter, "github-owner-filter", "", "GitHub org/user repository filter")
 	flag.StringVar(&gitHubTopicFilter, "github-topic-filter", "", "GitHub topic repository filter")
 
+	flag.StringVar(&S3Region, "aws-region", "", "AWS region such as us-east-1")
+	flag.StringVar(&S3Bucket, "aws-s3-bucket", "", "AWS bucket name")
```

Comment:
Please also change the actual command line argument names to `s3-region` and `s3-bucket` as well.

Reply:
Possibly also the descriptions